go.sum

```diff
@@ -1,10 +1,26 @@
 github.com/Masterminds/squirrel v1.2.0 h1:K1NhbTO21BWG47IVR0OnIZuE0LZcXAYqywrC3Ko53KI=
 github.com/Masterminds/squirrel v1.2.0/go.mod h1:yaPeOnPG5ZRwL9oKdTsO/prlkPbXWZlRVMQ/gGlzIuA=
```

Comment:
ignore this file, it is automatically generated by `go mod` tool adding `prometheus/client_golang`